expected/pg_mon.out

```diff
@@ -1,5 +1,6 @@
 create extension pg_mon;
 create extension pg_stat_statements;
+set pg_mon.log_new_query = false;
```

Comment:
It would be actually nice to see some tests.

Reply:
done that, but can not really check if relevant lines are added or not added in log file. However it definitely improves code coverage. :)